CenteredCollectionView/Classes/CenteredCollectionViewFlowLayout.swift

```diff
@@ -70,6 +73,10 @@ open class CenteredCollectionViewFlowLayout: UICollectionViewFlowLayout {
 				let inset = (currentCollectionViewSize.height - itemSize.height) / 2
 				collectionView.contentInset = UIEdgeInsets(top: inset, left: 0, bottom: inset, right: 0)
 				collectionView.contentOffset = CGPoint(x: 0, y: -inset)
+                
+            default:
+                collectionView.contentInset = UIEdgeInsets(top: 0, left: 0, bottom: 0, right: 0)
+                collectionView.contentOffset = .zero
```

Comment:
The spacing here doesn't match the rest of the code. Would you mind making the stacking consistent with the rest of the file?